llvm/test/Transforms/MemCpyOpt/memcpy-tbaa.ll

```diff
@@ -0,0 +1,88 @@
+; RUN: opt < %s -passes=memcpyopt,dse -S -verify-memoryssa | FileCheck %s
```

Comment:
Can you trim the test case to the minimum and auto generate the check lines please?